// openai-sdk/dist/example/index.js
"use strict";
var __importDefault = (this && this.__importDefault) || function (mod) {
    return (mod && mod.__esModule) ? mod : { "default": mod };
};
Object.defineProperty(exports, "__esModule", { value: true });
var index_1 = __importDefault(require("../index"));
var init = index_1.default.init, chat = index_1.default.chat, nlp = index_1.default.nlp;
init({
    TOKEN: 'azbqqVcqPTKig79PsNObuc3OK1tbBh',
    EncodingAESKey: 'SrYwMWrlVhI7ZCZvkvlcfG0cx4Xsnh2zR4sTDGvn8Qk'
});
chat({
    username: "uid",
    msg: "你好吗"
}).then(function (res) {
    console.log('机器人返回:', res);
}, function (res) {
    console.log('reject res:', res);
}).catch(function (e) {
    console.log('error', e);
});
// 分词
nlp.tokenize({
    uid: "uid",
    data: {
        q: "我的家乡叫中国。"
    }
}).then(function (res) {
    console.log('词法分析返回：', res);
}, function (res) {
    console.log('reject res:', res);
}).catch(function (e) {
    console.log('error', e);
});
// 数字日期时间识别
nlp.ner({
    uid: "uid",
    data: {
        q: "帮我订两张后天上午的火车票"
    }
}).then(function (res) {
    console.log('数字日期时间识别返回：', res);
}, function (res) {
    console.log('reject res:', res);
}).catch(function (e) {
    console.log('error', e);
});
// 情感分析
nlp.sentiment({
    uid: "uid",
    data: {
        q: "恭喜小张脱单成功",
        mode: "6class"
    }
}).then(function (res) {
    console.log('情感分析返回：', res);
}, function (res) {
    console.log('reject res:', res);
})
    .catch(function (e) {
    console.log('error', e);
});
// 敏感词识别
nlp.sensitive({
    uid: "uid",
    data: {
        q: "楼主真垃圾，祝你早日死全家"
    }
}).then(function (res) {
    console.log('敏感词识别返回：', res);
}, function (res) {
    console.log('reject res:', res);
}).catch(function (e) {
    console.log('error', e);
});
